File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Bahasa C/PRAK604-2310817120009-AlysaArmelia.c
#include <stdio.h>
#include<string.h>

int main()
{
    char deret1[45];
    char deret2[45];
    
    scanf("%[^\n]\n", deret1);
    scanf("%[^\n]", deret2);

    int panjang_kolom1 = strlen(deret1);
    int panjang_kolom2 = strlen(deret2);

    if (panjang_kolom1 != panjang_kolom2)
    {
        printf("Panjang kalimat berbeda, pesan palsu\n");
        
        
        } else if (panjang_kolom1 == panjang_kolom2){
            
            int baris1;
            int baris2;
            int bintang = 0;
            int pagar = 0;

            for (baris1 = 0, baris2 = 0; baris1 <panjang_kolom1 && baris2 <panjang_kolom2; ++baris1, ++baris2) 
            {
                if (deret1[baris1]==' '  && deret2[baris2]==' ')
                {
                printf(" ");
                }
                
                else if (deret1[baris1] == deret2[baris2])
                {
                printf("*");
                bintang++;
                
                }else{
                printf("#");
                pagar++;
                
                }
            }
            
            printf("\n* = %d\n", bintang);
            printf("# = %d\n", pagar);
            
            if(bintang > pagar)
            {
                printf("Pesan Asli\n");
            }

            else if(bintang < pagar)
            {
                printf("Pesan Palsu\n");
            }
    }
return 0;
}

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Python/PRAK604-2310817120009-AlysaArmelia.py
deret1 = input()
deret2 = input()
deret3 = deret1

if (len(deret1) != len(deret2)):
    print ("Panjang kalimat berbeda, pesan palsu")
    exit(1)

baris = 0
bintang = 0
pagar = 0

while(baris < len(deret1)):
    if deret1[baris] == " " and deret2[baris] ==" ":
        deret3 = deret3[:baris]+' '+deret3[:baris+1]
    
    else:
        if deret1[baris] == deret2[baris]:
            deret3 = deret3[:baris]+'*'+deret3[:baris+1]
            bintang+=1

        elif deret1[baris] != deret2[baris]:
            deret3 = deret3[:baris]+'#'+deret3[:baris+1]
            pagar+=1
    
    print(deret3[baris], end="")
    baris+=1

print()
print('* =', bintang)
print('# =', pagar)

if bintang > pagar:
    print("Pesan Asli")

elif bintang < pagar:
    print("Pesan Palsu")